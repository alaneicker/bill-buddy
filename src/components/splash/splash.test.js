import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import { Splash } from '.';

describe('Splash', () => {

  let component;

  beforeEach(() => {
    component = shallow(
      <Splash
        splashTitle="Splash Page Title"
        splashSubtitle="Splash Page Subtitle"
      />
    );
  });

  it('Should render without errors', () => {
    expect(shallowToJson(component)).toMatchSnapshot();
  });
});
