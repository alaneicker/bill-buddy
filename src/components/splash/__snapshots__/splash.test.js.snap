// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Splash Should render without errors 1`] = `
<div
  className="splash-page"
>
  <div
    className="splash-page__content"
  >
    <img
      alt="image of a rocket"
      src="/assets/images/launch.svg"
    />
    <h1>
      Splash Page Title
    </h1>
    <h2>
      Splash Page Subtitle
    </h2>
  </div>
</div>
`;
